it/src/test/java/com/sonar/runner/it/MultimoduleTest.java

```diff
@@ -45,18 +44,12 @@ public void test_simplest_with_props_on_root() {
 
     orchestrator.executeBuild(build);
 
-    Resource rootProject = findResource("simplest-with-props-on-root");
-    assertThat(rootProject.getName()).isEqualTo("Simplest multi-module project with all properties set on the root project");
-    assertThat(rootProject.getVersion()).isEqualTo("1.2.3");
+    assertThat(getComponent("simplest-with-props-on-root").getName()).isEqualTo("Simplest multi-module project with all properties set on the root project");
 
     // Verify that we have the modules
-    Resource module1 = findResource("simplest-with-props-on-root:module1");
-    assertThat(module1.getName()).isEqualTo("module1");
-    assertThat(module1.getVersion()).isEqualTo("1.2.3");
+    assertThat(getComponent("simplest-with-props-on-root:module1").getName()).isEqualTo("module1");
 
-    Resource module2 = findResource("simplest-with-props-on-root:module2");
-    assertThat(module2.getName()).isEqualTo("module2");
-    assertThat(module2.getVersion()).isEqualTo("1.2.3");
```

Comment:
@henryju I agree with you, I've discovered this missing feature too late..
I've added a comment on the JIRA ticket : https://jira.sonarsource.com/browse/SONAR-7745?focusedCommentId=111202&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-111202